ext/dom/xpath.c

```diff
@@ -70,6 +70,11 @@ static void dom_xpath_ext_function_php(xmlXPathParserContextPtr ctxt, int nargs,
 		return;
 	}
 
+	if (UNEXPECTED(nargs == 0)) {
+		zend_throw_error(NULL, "Function name must be passed as the first argument");
+		return;
+	}
+
```

Comment:
Would the for loop below be more clear as:
```c
for (i = fci.param_count; i > 0; i--) {
```
?

Reply:
Agreed, I'll change it soon